docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+
+Using namespace-scoped brokers, however, enables the broker to be installed in each namespace without conflicting at the class and plan level. When creating a service instance, you specify either the external class or plan name, or provide the class or plan identifier. Service Catalog then resolves these in order to determine which broker it should issue the provision command to. When using namespace-scoped brokers and their associated resources, this resolution occurs within the namespace. That means that users in namespace `backend-team` and namespace `front-end-team` can have the their own broker registrations and provision requests will be issued to the correct broker.
+
+### Limiting Access to Plans
+
+There are often situations when not all services and plans should be available to all users. A cluster administrator may wish to only provide free plans to certain users or restrict the ability to provision very expensive services. Additionally, when developers are creating new services that are exposed by brokers, they want to be able to iterate on those services without exposing them to all users in a cluster.
+
+Service Catalog's cluster-scoped resources for brokers, services, and plans are not sufficient to implement access control to ensure that users have access only to the service and plans that they should. For these resources, application of RBAC is really centered around what is visible to them, but is not enforced when a provision request is issued. For example, a `CLusterRole` could be created to prohibit a given user or group to view classes and plans, but this can not be used at provisioning time to ensure that the user cannot create the resource. Namespace-scoped brokers, services and plans, however, can be effectively combined with Kubernetes role based access control and Service Catalog Catalog Restrictions in order to provide more granular control over service instance provisioning.
+
+## Enabling Namespace Scoped Broker Resources
+
+Currently, namespace-scoped broker resources are an alpha-feature of Service Catalog behind a feature flag. To start using these resources, you will need to pass an argument to the API Server when you install Service Catalog: `--feature-gates NamespacedServiceBroker=true`.
+
+If you are using Helm, this flag has been surfaced as a chart parameter:
+
+```console
+helm install svc-cat/catalog \
+   --name catalog \
+   --namespace catalog \
+   --set namespacedServiceBrokerEnabled=true
+```
+
+If you are also using RBAC, providing this option to the Helm install command will also create the needed RBAC resources.
```

Comment:
Since we mention it here, I'm wondering if someone may want to do more with those RBAC resources (i.e. add themselves or someone to a role) or if that's more of a FYI that our helm chart isn't broken.